prow/genjobs/pkg/util/errors.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2019 Istio Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
```

Comment:
Can you delete this package please? Nothing uses it aside from the genjobs package. Just move these functions into that package.

Reply:
I like the abstraction and cleanliness using sub-packages, but again if you are adamantly opposed I will change it. I would like to reuse this code for future CLI tools, so will probably move it out anyway.